e2e_test/batch/basic/to_jsonb.slt.part

```diff
@@ -0,0 +1,249 @@
+statement ok
+SET RW_IMPLICIT_FLUSH TO true;
+
+query T
+select to_jsonb('Fred said "Hi."'::text)
+----
+"Fred said \"Hi.\""
+
+query T
+select to_jsonb(row(42, 'Fred said "Hi."'::text))
+----
+{"f1": 42, "f2": "Fred said \"Hi.\""}
+
+query T
+SELECT to_jsonb('hello'::text);
+----
+"hello"
+
+query T
+SELECT to_jsonb('hello'::varchar);
+----
+"hello"
+
+query T
+SELECT to_jsonb(10::integer);
+----
+10
+
+query T
+SELECT to_jsonb(15.87::float);
```

Comment:
FYI `float` means `double precision` in PostgreSQL. The 32-bit one is named `real`.

Reply:
changed to real